WalletWasabi/Services/Terminate/TerminateService.cs

```diff
@@ -20,6 +22,21 @@ public TerminateService(Func<Task> terminateApplicationAsync)
 			TerminateApplicationAsync = terminateApplicationAsync;
 			AppDomain.CurrentDomain.ProcessExit += CurrentDomain_ProcessExit;
 			Console.CancelKeyPress += Console_CancelKeyPress;
+
+			if (RuntimeInformation.IsOSPlatform(OSPlatform.Windows))
+			{
+				SystemEvents.SessionEnding += Windows_SystemEvents_SessionEnding;
+			}
+		}
+
+		private void Windows_SystemEvents_SessionEnding(object sender, SessionEndingEventArgs e)
+		{
+			// This event will only be triggered if you run Wasabi from the published package. Use the packager with the --onlybinaries option.
+			Logger.LogInfo($"Process termination was requested by the OS, reason {e.Reason}");
```

Comment:
```suggestion
			Logger.LogInfo($"Process termination was requested by the OS, reason '{e.Reason}'.");
```

It's often useful to know what is and what is not part of variables in logging messages.